app/models/todo_list.rb

```diff
@@ -1,6 +1,7 @@
 class TodoList < ApplicationRecord
   belongs_to :user
   has_many :todos, dependent: :destroy
+  has_many :collaborators, dependent: :destroy
 
```

Comment:
think of adding one more relation so that you can directly fetch collaborated todolists